docs/rfcs/033-merged-compute-image.md

```diff
@@ -0,0 +1,54 @@
+## Merged compute image
+https://github.com/neondatabase/neon/issues/6685
+
+### Motivation:
+It's hard to manage compute pools for 3 Postgres versions.  
+(we have a compute image for each version of Postgres (currently, it's 3 for neonVM and 3 for k8s pods; eventually, we will have only neonVMs)).
+We can try putting all Postgres versions into a single image, which should dramatically improve pool usage.
+
+### TODO
+#### Compute code changes:
+
+1. Create merged compute image https://github.com/neondatabase/neon/pull/6808
+2. Pass compute version in spec from control-plane
+3. Change path to the postgres in compute_ctl. Now it is not specified explicitly.
+   `compute_ctl` has `pgbin` and `pgdata` arguments, now they are used only in tests.
+3.  Make changes to custom_extension code - fix path handling.
+
+#### Control-plane changes:
+1. Pass compute version in spec from control-plane
+2. Remove old logic of VM pools management
```

Comment:
As Muhammet mentioned, I don't expect any critical obstacles here, but let's try to think about some approx. rollout plan.

Once universal compute image is ready we can:

1. Start always passing PG version in the spec, even though it's not yet used
2. Start using in in compute_ctl (won't change anything yet, as images are still PG-version-specific)
3. Make PG version optional in the pool configuration, which mean universal image will be used
4. Add test universal pools
5. Change forecasting service to predict cumulative pool size as well
6. Add feature flag to use universal computes
7. Rollout to some subset of users, run tests
8. Rollout to everyone, downscale old pools along the way
9. Tear down old pools completely

@mtyazici @lubennikovaav does it make sense to you? Anything else?